data/index.html

```diff
@@ -133,40 +133,52 @@ <h5 class="text-center">USGS Topo Maps</h5>
-    </div>
     <div class="grid__col section">
-        <h2>The Open SGID</h2>
-        <p class="readable">The Open SGID is a cloud-hosted PostGIS database containing the datasets from the SGID which are also available from the <a href="http://opendata.gis.utah.gov/">AGRC Open Data</a>. It is a publicly-available, direct database connection that can be utilized on a multitude of GIS platforms to access Utah spatial data. A <a href="https://github.com/agrc/open-sgid">GitHub repository</a> is used as the primary source of information for the Open SGID, and to catalog and address issues with explanations and or troubleshooting techniques. Please read, understand, and abide by the <a href="{% link about/policy/open-sgid/index.md %}">terms of use</a> prior to connecting.</p>
-        <p class="section-read-more">
-            <a href="{% link sgid/open-sgid/index.md %}" class="btn">Get connected</a>
+        <h2>Welcome to the State Geographic Information Database (SGID)</h2>
+        <p class="readable">The <a href="{% link sgid/index.html %}">SGID</a> aims to be a centralized repository of GIS datasets created or curated by state agencies. SGID datasets live in many different places depending on their type, availability, and popularity:
+        <ul class="readable"> 
+            <li>The curated Open SGID database</li>
+            <li>ArcGIS Online hosted feature services</li>
+            <li>SGID Open Data site</li>
+            <li>Raster Discovery web app</li>
+            <li>Discover basemap and imagery services</li>
+            <li>The AGRC Web API</li>
+        </ul>
         </p>
-    </div>
-    <div class="grid__col section alternate">
-    <h2>SGID Index</h2>
-        <div class="grid__col--span-all grid__col--centered grid__col--s-hidden" style="width:568px">
-            <a href="{% link data/sgid-index/index.html %}">
-                <img alt="SGID Index Table Search" class="text-center" src="{% link images/sgid-index.png %}">
-            </a>
-        </div>
-        <p class="readable">The SGID Index is a sortable and filterable table containing data items contained in our SGID. It is a great resource to quickly find GIS data and resources important to you. In the SGID Index, you will find the ISO category, the name of the resource, the agency who is the data steward, a short description and a link to a service endpoint if one is available. </p>
+
+        <p class="readable">The <a href="{% link data/sgid-index/index.html %}">SGID Index</a> is the master data index, regardless of where the data are stored or how they are served. It includes links to AGRC-maintained datasets as well as links to the authoritative sources for datasets created and served by other state agencies. Clicking on the category icons on this page will take you to the associated category in the Index.</p>
```

Comment:
**suggestion:** I realize the icons are both above and below this section, but perhaps use a link to direct readers to the Data Categories section -  _Clicking on the [category icons](https://deploy-preview-1508--gis-utah-gov.netlify.app/data/#data-categories) on this page..._

Reply:
Aha! I finally figured out how to link to headers. Done and done.